_posts/2023-03-06-NGINX-Dashboards-with-Fluent-Bit.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "Simple NGINX dashboards with Fluent Bit and OpenSearch"
+authors:
+  - jdbright
+  - lioperry
+  - anurag_gup
+  - armarble
+date:   2023-03-06
+categories:
+  - technical-post
+meta_keywords: opensearch simple schema, observability schema, opensearch opentelemetry, OpenSearch 2.6
+meta_description: While users can currently send their logs to OpenSearch, there are no set formats or schemas, which can make sharing dashboards and alerts cumbersome. In this blog post, we talk about using Fluent Bit, a new simple schema, and OpenSearch with NGINX as the first example.
+---
+
+Fluent Bit is a graduated sub-project under the Cloud Native Computing Foundation (CNCF) Fluentd project umbrella. Fluent Bit integrates with hundreds of common tools such as Kafka, Syslog, Loki, and, of course, OpenSearch.
+
+While users can currently send their logs to OpenSearch, there are no set formats or schemas for the logs, which can make sharing dashboards and alerts cumbersome. In this blog post, we talk about using Fluent Bit, a new simple schema, and OpenSearch with NGINX as the example.
+
+## Simple Schema for Observability
+
+OpenSearch 2.6 introduced a standardization for conforming to a common and unified observability schema: Simple Schema for Observability. 
+
+Observability is a collection of plugins and applications that let you visualize data-driven events by using PPL/SQL/DQL to explore and query data stored in OpenSearch. With the schema in place, Observability tools can ingest, automatically extract, and aggregate data and create custom dashboards, making it easier to understand the system at a higher level.
+
+Simple Schema for Observability is inspired by [OpenTelemetry](https://opentelemetry.io/docs/) and Elastic Container Service ([ECS](https://github.com/elastic/ecs)) logs categorization and field sets and OpenTelemetry metadata.
+
+## Connecting Simple Schema for Observability and Fluent Bit
+
+When raw logs are ingested by Fluent Bit, they are automatically converted into MessagePack, a binary representation of JSON. Fluent Bit’s automatic conversion makes parsing and modifying messages in-transit simpler.
+
+In addition, Fluent Bit comes with out-of-the-box parsers for common applications such as NGINX, Apache Web Logs, Kubernetes, and more. These parsers give structure to a log file, though they do not have the depth that other log schemas and formats do. "Fluent Bit can do more advanced transformations with things called processors (or filters) and even with a programming language called Lua."
```

Comment:
Why is the last sentence in quotes? Also, the last sentence does not connect to the text that comes before it. It needs to be evident why we're including the sentence where we're including it. What does the reader need to know and why?

Reply:
I did some shuffling around of the sentences here that I think ties it together and makes more sense. See what you think.